### ISSUE TITLE
No styles when running demo

### ISSUE THREAD

**Issue by varya** [lynndylanhurley/redux-auth-demo#4]
I followed the instructions to run the demo but got this.

<img width="1280" alt="screen shot 2016-02-22 at 21 50 14" src="https://cloud.githubusercontent.com/assets/27215/13230523/62c708a4-d9ae-11e5-80fa-d4e3ef5509c8.png">

It looks like styles are partly missing.

The environment is:

```
  › node -v
v4.2.2
  › npm -v
3.7.3
```

**Comment by laeg** [comment]
I think #2 is related:

Clone repo, install, `npm run watch` spits out the following error:

[1] /Users/laeg/Projects/React/redux-auth-demo/node_modules/redux-auth/bootstrap-theme.js:2
[1] ype:"submit",className:"pull-right",icon:this.props.icon,disabled:n,className:
[1]                                                                     ^^^^^^^^^
[1] SyntaxError: Duplicate data property in object literal not allowed in strict mode
[1]     at exports.runInThisContext (vm.js:73:16)
[1]     at Module._compile (module.js:443:25)
[1]     at Module._extensions..js (module.js:478:10)
[1]     at Object.require.extensions.(anonymous function) [as .js](/Users/laeg/Projects/React/redux-auth-demo/node_modules/babel-register/lib/node.js:138:7)
[1]     at Module.load (module.js:355:32)
[1]     at Module._load (module.js:310:12)
[1]     at Function.module._load (/Users/laeg/Projects/React/redux-auth-demo/node_modules/piping/lib/launcher.js:32:16)
[1]     at Module.require (module.js:365:17)
[1]     at require (module.js:384:17)
[1]     at Object.<anonymous> (/Users/laeg/Projects/React/redux-auth-demo/src/views/partials/RequestTestButton.js:17:23)

commenting the following lines [1229, 1230]

`className: "pull-right",
 icon: this.props.icon,`

Allows for the site to be displayed and renders the styling

**Comment by tylercollier** [comment]
I think this has something to do with the webpack bundle being 15 MB (huge). The first time I ran `npm run watch`, it took 19 seconds to compile it all. It took maybe 10 seconds for the styles to load in the browser, even though the HTML showed up right away. When I tried a different time (after a reboot, so everything had been reset in between), I let it sit again for 30 seconds and no styles loaded, but after pressing refresh, they showed up right away. I'm sure you tried refresh, but I'm just mentioning my experience.
